TranslatorReasonerAPI.yaml

```diff
@@ -1,14 +1,26 @@
 openapi: 3.0.1
 info:
   description: OpenAPI for NCATS Biomedical Translator Reasoners
-  version: 1.0.0
+  version: INSERT-VERSION-OF-YOUR-SERVICE-HERE
   title: OpenAPI for NCATS Biomedical Translator Reasoners
   contact:
     email: edeutsch@systemsbiology.org
   license:
     name: Apache 2.0
     url: http://www.apache.org/licenses/LICENSE-2.0.html
   termsOfService: INSERT-URL-HERE
+  x-trapi:
+    version: 1.1.0
+  x-translator:
+    component: INSERT-EITHER-ARA-OR-KP-HERE
```

Comment:
Team is not correctly structured here, and the component description doesn't include all the actual allowed values. See #217 for my recommendations of how to specify x-trapi and x-translator.